cluster-provision/k8s/1.18/manifests/cni.diff

```diff
@@ -0,0 +1,31 @@
+--- calico.3.12.0.yaml	2020-07-23 11:14:17.000000000 +0300
```

Comment:
This old name does not match the sources we have, does it? Shouldn't it match our cni.yaml?

Reply:
it doesn't need to match it applies anyway according the args give to patch commands
i generate the patch using diff -Naur between src and dst

Reply:
https://github.com/kubevirt/kubevirtci/pull/422/commits/29fe0fa0a11af07935ad687135dfaf63c7ca7972
here it changed, it depends on the file names i had locally when creating the diff
i dont think we should be strict about it